generators/javascript/logic.ts

```diff
@@ -63,7 +63,7 @@ export function logic_compare(
   block: Block,
   generator: JavascriptGenerator,
 ): [string, Order] {
-  // Dictionary of OP comparison operators and their implementations.
+  // Comparison operator.
```

Comment:
Is this a useful comment?

Reply:
The "Comparison operator" comment, which is here being restored having been inadvertently removed in a previous PR, is in a standard format that is common to almost all the generator functions.

Admittedly it should originaly have been a `/** */` JSDoc comment on the line above the `function` declaration.  I didn't want to try to fix all of them in this PR, so I've added that as a to-do item to #7600.

Reply:
Ohhh I didn't realize that this one was also a block-level comment. I thought it was about the dictionary itself.